src/index.ts

```diff
@@ -13,84 +13,63 @@ export function instantMeiliSearch(
 ): InstantMeiliSearchInstance {
   return {
     client: new MeiliSearch({ host: hostUrl, apiKey: apiKey }),
-    attributesToHighlight: ['*'],
     paginationTotalHits: options.paginationTotalHits || 200,
     primaryKey: options.primaryKey || undefined,
     placeholderSearch: options.placeholderSearch !== false, // true by default
     hitsPerPage: 20,
+    page: 0,
+
     /*
-      REQUEST CONSTRUCTION
+      REQUEST
     */
+
     transformToMeiliSearchParams: function ({
       query,
       facets,
       facetFilters,
       attributesToSnippet,
       attributesToRetrieve,
+      attributesToHighlight,
       filters,
     }) {
-      const limit = this.pagination // if pagination widget is set, use paginationTotalHits as limit
-        ? this.paginationTotalHits
-        : this.hitsPerPage
+      const limit = this.paginationTotalHits
+      const attributesToCrop = attributesToSnippet
```

Comment:
If you really want to rename `attributesToSnippet` to `attributesToCrop`, you can do it directly in the deconstruction, and avoid creating a new variable :) 

```js
    transformToMeiliSearchParams: function ({
      query,
      facets,
      facetFilters,
      attributesToSnippet: attributesToCrop,
      attributesToRetrieve,
      attributesToHighlight,
      filters,
    }) {
```